pageStyles/candidates.module.scss

```diff
@@ -0,0 +1,43 @@
+﻿@import 'pageStyles/constants';
+
+.link {
+  margin: unset;
+}
+
+.desktopButton {
```

Comment:
There's quite a lot in common between these 2 - if we don't turn it into 1 button, we should try to commonise these a little.

Reply:
One button now